src/Compilers/Core/Portable/Compilation/SymbolInfo.cs

```diff
@@ -14,7 +14,7 @@ public struct SymbolInfo : IEquatable<SymbolInfo>
     {
         internal static readonly SymbolInfo None = new SymbolInfo(null, ImmutableArray<ISymbol>.Empty, CandidateReason.None);
 
-        private ImmutableArray<ISymbol> _candidateSymbols;
+        private readonly ImmutableArray<ISymbol> _candidateSymbols;
```

Comment:
Why the readonly changes here? Did some analyzer get updated and is flagging this now?

Reply:
Yes. :)